### PR TITLE
[NEAT-218] 🐻‍❄️ Ensure Lifecycle Tutorials are up-to-date

### PR DESCRIPTION
**Pull request by doctrino**
* Read through existing parts and fixed typos and incorrect information
* Updated artifacts (added dataModelType to information sheets)
* Added missing configs to Steps.
* Updated outdated screenshots

### CODE REVIEW

File: cognite/neat/rules/models/information/_rules.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ class InformationMetadata(BaseMetadata):
             "typically information architects are considered as contributors."
         ),
     )
+    license: str | None = None
+    rights: str | None = None
```
Comment (doctrino) [inline cognite/neat/rules/models/information/_rules.py:103]: These are in the tutorial but actually not read by the information rules, I added them here as we should at least read them. Alternative is to remove them from the tutorial.

Comment (nikokaoja) [reply cognite/neat/rules/models/information/_rules.py:103]: Great !

File: cognite/neat/workflows/steps/lib/current/rules_importer.py [hunk 2]
```diff
@@ -184,6 +185,13 @@ class DMSToRules(Step):
             type="string",
             required=True,
         ),
+        Configurable(
```
Comment (nikokaoja) [inline cognite/neat/workflows/steps/lib/current/rules_importer.py:188]: super nice!

File: docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md [hunk 3]
```diff
@@ -20,13 +20,23 @@ workflow, and then clicks on the `Import DMS` step. This opens the modal with th
 Olav selects the following options:
 
 * **Data model id**: This is the id of the enterprise model. Olav finds this ID by login into CDF.
+* **Reference data model id** This can be ignored for now. This is used when you want to update a solution
+  model to also download the enterprise model the solution model is built on.
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md:24]:
```suggestion
  model to also download the enterprise model that the solution model is built on.
```

File: docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md [hunk 4]
```diff
@@ -20,13 +20,23 @@ workflow, and then clicks on the `Import DMS` step. This opens the modal with th
...
-  true as the enterprise model is a reference model. This means that the enterprise model will not be changed
-  and that the solution model will be built on top of the enterprise model.
+
+Furthermore, Olav clicks on the `Create Excel Sheet` step which opens a modal with the configuration for the export
+
+<img src="../../artifacts/figs/life_cycle_download_reference_model_analytic_solution_model_export.png" height="300">
+
+* **Styling**: `maximal`. This is how the exported Excel document is styled, Olav selects `maximal` as this is
+  makes the model easier to read.
+* **Output role format**: `intput`. This is the same as the role format in the `Import DMS` step. Olav
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md:36]:
```suggestion
* **Output role format**: `input`. This is the same as the role format in the `Import DMS` step. Olav
```

I am wandering should we rename `input` to `original` ?

File: docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md [hunk 5]
```diff
@@ -55,22 +65,23 @@ Olav starts by setting up the metadata for the solution model. He opens the `Met
...
-
-The most important part of the metadata sheet is the `prefix` and `schema`. The `prefix` is used differentiate
-the solution model from the enterprise model. If Olav set the `prefix` to the same as the enterprise model, this
-would mean that this is an extension of the enterprise model and not a new model. The `schema` is used to
+|               |                                                       |
+|---------------|-------------------------------------------------------|
+| role          | information architect                                 |
+| creator       | Olav                                                  |
+| dataModelType | solution                                              |
+| namespace     | http://purl.org/cognite/power_analytic                |
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md:73]:
```suggestion
| namespace     | http://purl.org/cognite/power_analytic/                |
```

the namespace must end with either "/" or "#" , I think we do magically fix this (at least we were doing this in legacy)

File: docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md [hunk 6]
```diff
@@ -55,22 +65,23 @@ Olav starts by setting up the metadata for the solution model. He opens the `Met
...
+| prefix        | power_analytic                                        |
+| schema        | extended                                              |
+| created       | 2024-03-26                                            |
+| updated       | 2024-02-09                                            |
+| version       | 0.1.0                                                 |
+| title         | Power Forecast Model                                  |
+| description   | Solution model for WindFarm power production forecast |
+
+The most important part of the metadata sheet is the `dataModelType`, `prefix` and `schema`. The `dataModelType`
+is used to tell neat what type of model this is, which has impact on the validation. The `prefix` is should be
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md:83]:
```suggestion
is used to tell neat what type of model this is, which has impact on the validation. The `prefix` and `namespace` should be
```

File: docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md [hunk 7]
```diff
@@ -55,22 +65,23 @@ Olav starts by setting up the metadata for the solution model. He opens the `Met
...
+different from the enterprise model, as each data model should have its own namespace. The `schema` is used to
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md:84]:
```suggestion
different from the enterprise model, as each data model should have unique prefix and namespace. The `schema` is used to
```

File: docs/tutorials/data-modeling-lifecycle/part-3-extending-enterprise-model.md [hunk 8]
```diff
@@ -101,38 +104,42 @@ The downloaded spreadsheet contains six sheets:
...
-  against.
-* **RefMetadata**: (READ ONLY) This will be the metadata from the current Enterprise model.
+* **LastProperties**: (READ-ONLY) This will be all the properties from the enterprise model that Svein Harald can use to
+  look up what properties he wants to use in the solution model. In addition, this will be used in the validation
+  to ensure that the new changes do break the existing model.
+* **LastClasses**: (READ-ONLY) This will be all the classes from the current enterprise model. Similar to the `LastProperties`,
+  this will be used to look up, and will be validated against.
+* **LastMetadata**: (READ-ONLY) This will be the metadata from the current Enterprise model.
+
+**Node** The **Last** sheets are used by **NEAT** to validate against depending on the `extension` in the `Metadata`
```
Comment (nikokaoja) [inline docs/tutorials/data-modeling-lifecycle/part-3-extending-enterprise-model.md:114]:
```suggestion
**Note** The **Last** sheets are used by **NEAT** for validations, which are dependent the `extension` configuration in the `Metadata`
```

**Review by nikokaoja — APPROVED** [review]

**Pull request merged**